process_jartic_archives_fast.py

```
@@ -0,0 +1,217 @@
+        cleanup_after_parse=False
+    )
+    
+    print("\n📍 Finding locations...")
+    async with datasource:
+        locations = await datasource.find_locations(country_code="JP")
+        if args.max_locations:
+            locations = locations[:args.max_locations]
+    print(f"   Found {len(locations)} locations")
+    
+    # Prepare output
+    timestamp = datetime.now().strftime('%Y%m%d_%H%M%S')
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)